src/core/ApplicationDescription.cpp

```diff
@@ -25,6 +25,11 @@
 #include "JsonHelper.h"
 #include "LogManager.h"
 
+// The size from AGL Window.
+// https://jira.automotivelinux.org/browse/SPEC-1890
+constexpr unsigned int kWebAppWidth = 608;
+constexpr unsigned int kWebAppHeight = 837;
```

Comment:
I'd prefer we'd avoid hardcoding this here. If we are not getting any resolution for platform, at least the value should be added in the AGL files.

The solution could be just provide the resolution from environment variable (i.e. WAM_DEFAULT_OVERRIDE_HEIGHT and WAM_DEFAULT_OVERRIDE_WIDTH), and use the values in the WebAppWaylandAGL to set the overrides in case app does not provide its own override.

BTW, is that resolution of 608x837 provided by the application manifest? I guess the answer is no, but...

I'd prefer that we'd add those env vars from wam.bb script, and it would be even better if we could know the values at yocto level instead of hardcoding them even in the script.